src/functions.ts

```diff
@@ -30,7 +30,7 @@ async function getAthleteStats(token_access: string, id_atleta: string){
     }
   }
 
-  const response = await fetch(`https://www.strava.com/api/v3/athletes/${id_atleta}/stats`,config)
+  const response = await fetch(`${process.env.STRAVA_BASE_URL}/athletes/${id_atleta}/stats`,config)
```

Comment:
Nel caso in cui la variabile d'ambiente STRAVA_BASE_URL non fosse presente qui avremmo un errore.

Nonostante ciò credo sia scomodo effettuare i controlli ogni volta che dobbiamo utilizzare le variabili d'ambiente, secondo me la soluzione migliore è aggiungere un controllo per tutte le variabili d'ambiente nel momento in cui viene avviata l'applicazione, lo faremo!

Reply:
ok mettiamolo come task da fare insieme magari

Reply:
Si, adesso creo un issue. Lo assegno a entrambi poi ne parliamo giovedì.